crates/transaction-pool/src/test_utils/mock.rs

```diff
@@ -541,6 +551,8 @@ impl FromRecoveredTransaction for MockTransaction {
                 gas_limit,
                 to,
                 value: value.into(),
+                accesslist: todo!(),
+                input: todo!(),
```

Comment:
we need to use the values from Line 543-544

Reply:
> we need to use the values from Line 543-544

Hello again @mattsse i'm really sorry i was in school. I'm back now. I still don't get the idea on how to implement this part. Could you please if you have time tell me more about it ? Im sorry if my questions can be stupid